resources/views/Widgets/Common/ImageBoxWidget.twig

```diff
@@ -5,29 +5,28 @@
 {% set style            = widget.settings.style.mobile %}
 {% set headline         = widget.settings.headline.mobile %}
 {% set imageSize        = widget.settings.imageSize.mobile %}
-{% set categoryId       = widget.settings.categoryId.mobile %}
-{% set variationId      = widget.settings.variationId.mobile %}
-{% set customImagePath  = widget.settings.customImagePath.mobile %}
+{% set urlPickerType    = widget.settings.urlPicker.type.mobile %}
+{% set urlPickerValue   = widget.settings.urlPicker.value.mobile %}
 
 {{ Twig.set("name", "" | json_encode) }}
 {{ Twig.set("url", "" | json_encode) }}
 {{ Twig.set("imageUrl", "" | json_encode) }}
 
-{% if categoryId | trim is not empty %}
+{% if urlPickerType == 'category' %}
```

Comment:
Da die Komponente auch in anderen Widgets verwendet werden wird, würde ich diese Logik in ein Twig-Marko auslagern.